src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -146,9 +146,9 @@
             :passwords-match?          same-passwords?
             :empty-password?           empty-password?
             :show-password-validation? @show-password-validation?
-            :on-input-focus            (fn [input-id]
+            :on-input-focus            (fn []
                                          (scroll-to-end-fn)
-                                         (reset! focused-input input-id))
+                                         (reset! focused-input :password))
```

Comment:
@ulisesmac , As you mentioned since we don't need to `:repeat-password`, i have removed the parameter in `:on-input-focused` and defaulted the `input-id` to `:password`.

Is this change okay?

Reply:
Yes, it's ok! thank you! :+1: